phpunit.xml

```diff
@@ -29,8 +26,14 @@
         <env name="DB_DATABASE" value="apptest"/>
         <env name="DB_HOST" value="mysql-test"/>
         <env name="MAIL_MAILER" value="array"/>
-        <env name="QUEUE_CONNECTION" value="sync"/>
+        <env name="QUEUE_CONNECTION" value="null"/>
         <env name="SESSION_DRIVER" value="array"/>
         <env name="TELESCOPE_ENABLED" value="false"/>
+        <ini name="memory_limit" value="256M"/>
     </php>
+    <source>
```

Comment:
Is this required? I feel like on that same other PR that a section like this was removed - but I can't remember.